blog/_posts/2023-04-20-sbt-plugins-community-repository.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy, Scala Center
+title: "sbt Plugins Community Repository"
+---
+
+The 7th of April 2023, many Scala builds stopped working because an outage affected the sbt community repository.
+In this blog post, we explain what the sbt community repository is and what it stores, we report on the incident
+that happened, and we outline what we can do to avoid such problems in the future.
```

Comment:
```suggestion
In this blog post, we explain what the sbt community repository is and what it stores, report on the incident
that happened, and outline what we can do to avoid such problems in the future.
```

Reply:
you can also omit "that happened", that is inherent to being an incident :-)